src/Feature/Account/client/ChangePassword/index.ts

```diff
@@ -42,6 +45,6 @@ const mapDispatchToProps = (dispatch: Dispatch) =>
 const connectedToStore = connect<ChangePasswordStateProps, ChangePasswordDispatchProps, ChangePasswordOwnProps>(
   mapStateToProps,
   mapDispatchToProps
-)(Component);
+);
 
-export const ChangePassword = withExperienceEditorChromes(connectedToStore);
+export const ChangePassword = compose(withExperienceEditorChromes, connectedToStore, renderingWithContext)(Component) ;
```

Comment:
Remove `withExperienceEditorChromes` because `renderingWithContext` include it

Reply:
done